example-app/SantokuApp/src/features/demo-react-query/pages/DisableErrorHandlerDemoPage.tsx

```diff
@@ -12,14 +12,14 @@ const onError = () => {
 };
 
 export const DisableErrorHandlerDemoPage: React.FC = () => {
-  const query1 = useQuery(['dummy1'], queryErrorFn, {
+  const query1 = useQuery(['dummy1', 'disableErrorHandler1'], queryErrorFn, {
```

Comment:
↓とかの方が良さそう？？
```suggestion
  const query1 = useQuery(['disableErrorHandler', 'queryErrorFn1'], queryErrorFn, {
```

Reply:
こっちの方が良いですかね？
```suggestion
  const query1 = useQuery(['disableErrorHandler', 'queryErrorFn', {enabled: false}], queryErrorFn, {
```

Reply:
第2引数は`queryErrorFn1`のようにする必要があると思いますが、認識あってますでしょうか。
```suggestion
  const query1 = useQuery(['disableErrorHandler', 'queryErrorFn1', {enabled: false}], queryErrorFn, {
```

Reply:
いえ、`queryErrorFn`までは同じで、3つ目の指定だけ変えても良いかなぁと思っていました。

Reply:
引数名を変更しました。
対応Commit: f96de75aee3f86674a66db901a33c9d23b734bd3